src/app.ts

```diff
@@ -7,6 +7,7 @@ import errorHandler from "./controllers/error.controller";
 import { serviceAvailabilityMiddleware } from "./middleware/service.availability.middleware";
 import { authenticationMiddleware } from "./middleware/authentication.middleware";
 import { sessionMiddleware } from "./middleware/session.middleware";
+import { companyNumberQueryParameterValidationMiddleware } from "./middleware/company.number.validation.middleware";
```

Comment:
V minor, but could move this import next to the other query param validation import command on line 14.